scripts/badger/registry.py

```diff
@@ -2,20 +2,99 @@
 from great_ape_safe import GreatApeSafe
 from helpers.addresses import get_registry as registry
 
-console = Console()
+C = Console()
 
 # assuming techOps
-contract_registry = registry()
-chain_safe = GreatApeSafe(contract_registry.badger_wallets.techops_multisig)
-chain_safe.init_badger()
+r = registry()
+SAFE = GreatApeSafe(r.badger_wallets.techops_multisig)
+SAFE.init_badger()
+
+# Allowed metadata behaviors
+# NOTE: Should be kept up to date matching the behaviors defined on the SDK at
+# https://github.com/Badger-Finance/badger-sdk/blob/main/src/vaults/enums/vault-behavior.enum.ts
+ALLOWED_BEHAVIORS = [
+  'Compounder',
+  'Helper',
+  'DCA',
+  'Ecosystem',
+  'Ecosystem Helper',
+  'None'
+]
+
+# Add any vaults's addresses that you want to promote or demote
+VAULTS_ARRAY = []
+# Add the new statuses you want to promote or promote each vault to
+NEW_STATUS_ARRAY = []
```

Comment:
this might make more sense as a dict, ie `{vault1: new_status, vault2: new_status}` etc. but minor detail ofc.